README.md

```diff
@@ -6,10 +6,11 @@ Using pyfakefs, your tests operate on a fake file system in memory without
 touching the real disk. The software under test requires no modification to
 work with pyfakefs.
 
-pyfakefs acts as a `pytest` plugin by providing the `fs` fixture, which will
-automatically invoke the fake filesystem. It also provides
-the `fake_filesystem_unittest.TestCase` class for use with `unittest` and
-the means to use the fake filesystem with other test frameworks.
+Pyfakefs creates a new empty in-memory file system at each test start, which replaces
+the real filesystem during the test. There are several means to achieve this: by using
```

Comment:
Maybe you could add an intuition-oriented line saying something like:

> Think of pyfakefs as making a per-test temporary directory, except for an entire file system.

Reply:
This way?

Reply:
Yeah I like it, thanks for your help!  Appreciate it